app/models/mixins/process_tasks_mixin.rb

```diff
@@ -1,3 +1,5 @@
+require 'manageiq-api-client'
```

Comment:
@carbonin Can you move this down to line 124 or so?  I'd prefer to see this delay loaded